src/server/pfs/server/master_v2.go

```diff
@@ -21,21 +21,24 @@ const (
 func (d *driverV2) master(env *serviceenv.ServiceEnv, objClient obj.Client, db *gorm.DB) {
 	ctx := context.Background()
 	masterLock := dlock.NewDLock(d.etcdClient, path.Join(d.prefix, masterLockPath))
-	err := backoff.RetryNotify(func() error {
-		masterCtx, err := masterLock.Lock(ctx)
-		if err != nil {
+	for {
```

Comment:
This outer for loop should be unnecessary. If we get in this situation, then there is a bug elsewhere that is causing us to return nil in the retry. I think we just want the panic if we ever exit the retry.